neutral_generation/constants.py

```diff
@@ -0,0 +1,67 @@
+# pronouns
+NON_FUNCTION_PRONOUNS = {   # need to change pronoun based on context
```

Comment:
I see! This is a great observation. I will look into implementing this. 

I think using the dependency structure will also improve runtime significantly since loading and using the LM is what takes the most time in our Python script.